app/src/main/java/com/xinto/opencord/domain/manager/PersistentDataManager.kt

```diff
@@ -4,52 +4,52 @@ import android.content.SharedPreferences
 import com.xinto.opencord.domain.manager.base.BasePreferenceManager
 
 interface PersistentDataManager {
-    var persistentGuildId: ULong
-    var persistentChannelId: ULong
-    var collapsedCategories: List<ULong>
-    fun addCollapsedCategory(id: ULong)
-    fun removeCollapsedCategory(id: ULong)
+    var persistentGuildId: Long
+    var persistentChannelId: Long
+    var collapsedCategories: List<Long>
+    fun addCollapsedCategory(id: Long)
+    fun removeCollapsedCategory(id: Long)
 }
 
 class PersistentDataManagerImpl(
     persistentPrefs: SharedPreferences
 ) : BasePreferenceManager(persistentPrefs), PersistentDataManager {
-    override var persistentGuildId: ULong
-        get() = getLong(CURRENT_GUILD_ID_KEY, CURRENT_GUILD_ID_DEFAULT).toULong()
+    override var persistentGuildId: Long
+        get() = getLong(CURRENT_GUILD_ID_KEY, CURRENT_GUILD_ID_DEFALT).toLong()
         set(value) {
             putLong(CURRENT_GUILD_ID_KEY, value.toLong())
         }
 
-    override var persistentChannelId: ULong
-        get() = getLong(CURRENT_CHANNEL_ID_KEY, CURRENT_CHANNEL_ID_DEFAULT).toULong()
+    override var persistentChannelId: Long
+        get() = getLong(CURRENT_CHANNEL_ID_KEY, CURRENT_CHANNEL_ID_DEFALT).toLong()
         set(value) {
             putLong(CURRENT_CHANNEL_ID_KEY, value.toLong())
         }
 
-    override var collapsedCategories: List<ULong>
+    override var collapsedCategories: List<Long>
         get() = getStringSet(COLLAPSED_CATEGORIES_ID_KEY, emptySet())!!
-            .mapNotNull { it.toULongOrNull() }
+            .mapNotNull { it.toLongOrNull() }
         set(value) {
             putStringSet(
                 COLLAPSED_CATEGORIES_ID_KEY,
                 value.map { it.toString() }.toSet()
             )
         }
 
-    override fun addCollapsedCategory(id: ULong) {
+    override fun addCollapsedCategory(id: Long) {
         collapsedCategories = collapsedCategories + id
     }
 
-    override fun removeCollapsedCategory(id: ULong) {
+    override fun removeCollapsedCategory(id: Long) {
         collapsedCategories = collapsedCategories - id
     }
 
     companion object {
         const val CURRENT_GUILD_ID_KEY = "current_guild_id"
-        const val CURRENT_GUILD_ID_DEFAULT = 0L
+        const val CURRENT_GUILD_ID_DEFALT = 0L
 
         const val CURRENT_CHANNEL_ID_KEY = "current_channel_id"
-        const val CURRENT_CHANNEL_ID_DEFAULT = 0L
+        const val CURRENT_CHANNEL_ID_DEFALT = 0L
```

Comment:
Looks like you did a faulty search and replace here for UL

Reply:
i like this spelling better

Reply:
AHAHAHAHHAH LMAO